azurerm/internal/services/loganalytics/log_analytics_saved_search_resource.go

```diff
@@ -212,13 +214,14 @@ func resourceArmLogAnalyticsSavedSearchDelete(d *schema.ResourceData, meta inter
 	client := meta.(*clients.Client).LogAnalytics.SavedSearchesClient
 	ctx, cancel := timeouts.ForDelete(meta.(*clients.Client).StopContext, d)
 	defer cancel()
-	id, err := parse.LogAnalyticsSavedSearchID(d.Id())
+	// FIXME: @favoretti: API returns ID without a leading slash
+	id, err := parse.LogAnalyticsSavedSearchID(fmt.Sprintf("/%s", strings.TrimPrefix(d.Id(), "/")))
```

Comment:
as a temporary hack this should be fine, we'll add a state migration for the future